### ISSUE TITLE
[feature request] More descriptive diffs for merge commits

### ISSUE THREAD

**Issue by SirIntruder** [gitextensions/gitextensions#3709]
In merge commits, you currently see:
```
---diff with parent: [commit hash a]
X
Y
---diff with parent: [commit hash b]
X
Y
Z
---combined diff
X
```

Parent commit hashes do not mean much, can we do something to make this more descriptive? Like highlight parent commit when diff is selected? Simply show commit message next to hash?

**Comment by 62mkv** [comment]
why is this "closed" ? is it implemented and will be rolled out in 2.51 ?

**Comment by RussKie** [comment]
Yes, it is. Yes, it will be.
Please refer to https://github.com/gitextensions/gitextensions/commit/f0d50a53c8386dab510a6a7df10a2f752471047e
